### PR TITLE
Simple General Inclusions

### PR DESCRIPTION
**Pull request by apozharski**
Adding expert user functionality to manually define `alpha` and `f_x` to handle non-fillipov systems. 

Also added printing of objective.

### CODE REVIEW

**Review by nurkanovic — COMMENTED** [review]
LGTM, except some small verbose things. 
the alignment is a bit disturbed in the homotopy solver, at least in my command window.
When I copy it here it looks nice for some reason.
```

iter		sigma		compl_res	objective	CPU time	NLP iters	status
1		1.00e+00	2.10e+02	59.899		7.083		496		Solved_To_Acceptable_Level
```
quite  confusing...

Anyway, it seems to me that the homotopy solver still prints the $\ell_1$ cross comp and not $\ell_{\infty}$
In the final verbose after the problem is solve i would not print the $\ell_1$ anymore but just the $\ell_\infty$.

File: src/matlab/homotopy_solver.m [hunk 1]
```diff
@@ -63,15 +63,8 @@
...
-% TODO remove try!
-try
-    complementarity_stats = [full(comp_res(w0, p_val))];
-catch
-    w0 = w0(1:length(model.w));
-    w0
-    p_val
-    complementarity_stats = [full(comp_res(w0, p_val))];
-end
+complementarity_stats = [full(comp_res(w0, p_val))];
```
Comment (nurkanovic) [inline src/matlab/homotopy_solver.m:66]: comp_res still computes $\ell_1$ without the cross terms, right? 
nice that the stupid try-catch is resolved.

Comment (apozharski) [reply src/matlab/homotopy_solver.m:66]: Yep, it still does, I haven't quite finished that, but hopefully will today in a follow on PR. As for the formatting I made it look nice in my command window so it seems somehow our machines are handling tabs differently. I'm going to look if matlab has a nice table output formatter, and if it does switch to using that.

Comment (nurkanovic) [reply src/matlab/homotopy_solver.m:66]: Lets get the current PR through as the inclusion part is anyway good and let the verbose be part of the next one.

**Review by nurkanovic — APPROVED** [review]

**Pull request merged**